marshal.go

```diff
@@ -20,6 +20,7 @@ import (
 
 	"gopkg.in/inf.v0"
 
+	cqlint "github.com/gocql/gocql/marshal/int"
```

Comment:
is this cqlint needed, can you just use `int` similarly to `smallint` and `tinyint`?

Reply:
Because name `int` already busy

Reply:
so maybe rename the packe to cqlint to not have this thing here

Reply:
Good idea.

Reply:
Done